spec/rails_helper.rb

```diff
@@ -32,7 +34,6 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{:Rails.root}/spec/fixtures"
```

Comment:
- [ ] Hi @t-bello7, for some reason your tests were not passing probably cause by this line of code:
![Screenshot from 2022-12-15 06-05-46](https://user-images.githubusercontent.com/63087491/207777293-f358d034-7d3e-46a8-b5d0-02e048778e13.png)

- After I comment the line of code, the tests ran successfully:


![Screenshot from 2022-12-15 06-09-27](https://user-images.githubusercontent.com/63087491/207777531-03abd0e3-1e83-41ee-906b-d8755edebbdc.png)

Reply:
I get this error when I push to GitHub. Do you have a suggestion on how I can fix it?
![Screenshot from 2022-12-15 06-35-06](https://user-images.githubusercontent.com/24270269/207780911-f63676e1-a738-4279-a33e-767c82123ed5.png)